source/index.js

```diff
@@ -96,6 +95,13 @@ const np = async (input = 'patch', options, {pkg, rootDir}) => {
 	// To prevent the process from hanging due to watch mode (e.g. when running `vitest`)
 	const ciEnvOptions = {env: {CI: 'true'}};
 
+	/** @param {typeof options} _options */
+	function getPublishCommand(_options) {
+		const publishCommand = pkgManager.publishCommand || (args => [pkgManager.cli, args]);
```

Comment:
nit: [`prefer-nullish-coallescing`](https://typescript-eslint.io/rules/prefer-nullish-coalescing/)

```suggestion
		const publishCommand = pkgManager.publishCommand ?? (args => [pkgManager.cli, args]);
```

Could change these in a couple of places. Not a huge deal though.

Reply:
IMO `??` is only better if you would want to stick with `0` or `false` or `''` or whatever. Given this is supposed to be either a `Function` or `undefined`. I think that if someone passed `false` they would _definitely_ prefer to get the default function, and not have a runtime error at some unknown point in the future, `false is not a function`.

`??` is still useful for nullish booleans and numbers, it's just not better across the board, and in this case I think `||` is better.

I won't die on that hill though, if it'll block an approval I'll change it.

Reply:
I don't mind either way, just something I like covering my bases on. Since we don't provide a library I lean more towards being "correct," but I also haven't looked too close at the types from this PR in a week so I'll defer to you.

Reply:
Ok I'll leave like this unless there are more complaints!